OptimizelySDK.Tests/AudienceConditionsTests/ConditionEvaluationTest.cs

```diff
@@ -655,5 +656,67 @@ public void TestInvalidSemVersions()
         }
 
         #endregion
+        
+        #region Qualified Tests
+
+        private const string QUALIFIED = "qualified";
+        private const string CUSTOM_ATTRIBUTE = "custom_attribute";
+
+        [Test]
+        public void QualifiedConditionWithNonStringValueShouldFail()
+        {
+            var condition = new BaseCondition()
+            {
+                Type = CUSTOM_ATTRIBUTE,
```

Comment:
If it's not intentional error testing, let's change this to THIRD_PARTY_DIMENSION.

Reply:
Same to other tests below